notebooks/sen2_blade_csv_plans_placements.clj

```diff
@@ -137,13 +135,29 @@
                          sen2-blade-csv-plans-placements/plans-placements-on-census-dates-col-name->label)
 
 
+;;; ### Write plans & placements file
+^{::clerk/visibility {:code :hide}}
+(clerk/md (format "Write `%s`  \nto working directory: %s:" (tc/dataset-name plans-placements-on-census-dates) wk-dir))
+^{::clerk/visibility {:code :show, :result :hide}}
+(comment
+  (let [ds              plans-placements-on-census-dates
+        file-name-stem  (tc/dataset-name ds)
+        col-name->label sen2-blade-csv-plans-placements/plans-placements-on-census-dates-col-name->label]
+    (tc/write! (tc/dataset {:column-number (iterate inc 1)
+                            :column-name   (map name   (tc/column-names ds))
+                            :column-label  (map col-name->label (tc/column-names ds))})
+               (str wk-dir file-name-stem "-col-labels.csv"))
+    (tc/write! ds
+               (str wk-dir file-name-stem ".csv")))
```

Comment:
This feels like the important output and the handover to the next step, IIUC.

Reply:
UUC: I've added some explanation and code to read it back in as part of  the update step.